examples/gallery/plot/colorbar.py

```diff
@@ -2,51 +2,59 @@
 Colorbar
 --------
 
-The :meth:`pygmt.Figure.colorbar` method creates a color scalebar. We must specify the
-colormap via the ``cmap`` argument, and set the placement via the ``position`` argument.
-The full list of color paletted tables can be found at :gmt-docs:`cookbook/cpts.html`.
-You can set the `position` of the colorbar using the following options:
-
-- j/J: justified inside/outside the mapframe using any 2 character combination of
-  vertical (**T** op, **M** iddle, **B** ottom) and horizontal (**L** eft, **C** enter,
-  **R** ight) alignment codes, e.g. `position="jTR"` for top right.
-- g: using map coordinates, e.g. `position="g170/-45"` for longitude 170E, latitude 45S.
-- x: using paper coordinates, e.g. `position="x5c/7c"` for 5cm,7cm from anchor point.
+The :meth:`pygmt.Figure.colorbar` method creates a color scalebar. We must
+specify the colormap via the ``cmap`` argument, and set the placement via the
+``position`` argument. The full list of color palette tables can be found at
+:gmt-docs:`cookbook/cpts.html`. You can set the `position` of the colorbar
+using the following options:
+
+- j/J: justified inside/outside the mapframe using any 2 character combination
+  of vertical (**T** op, **M** iddle, **B** ottom) and horizontal (**L** eft,
+  **C** enter, **R** ight) alignment codes, e.g. `position="jTR"` for top
+  right.
+- g: using map coordinates, e.g. `position="g170/-45"` for longitude 170E,
+  latitude 45S.
+- x: using paper coordinates, e.g. `position="x5c/7c"` for 5cm,7cm from anchor
+  point.
 - n: using normalized (0-1) coordinates, e.g. `position="n0.4/0.8"`.
 
-Note that the anchor point defaults to the bottom left (BL). Append +h to ``position``
-to get a horizontal colorbar instead of a vertical one. For more advanced styling
-options, see the full option list at :gmt-docs:`colorbar.html`.
+Note that the anchor point defaults to the bottom left (BL). Append +h to
+``position`` to get a horizontal colorbar instead of a vertical one. For more
+advanced styling options, see the full option list at
+:gmt-docs:`colorbar.html`.
 """
 import pygmt
 
 fig = pygmt.Figure()
-fig.basemap(region=[0, 3, 6, 9], projection="t0/3c", frame=True)
+fig.basemap(region=[0, 3, 6, 9], projection="x3c", frame=["af", 'WSne+t"Colorbars"'])
 
-# Create a colorbar suitable for surface topography- oleron
+## Create a colorbar designed for seismic tomography- roma
+# Colorbar is placed at bottom center (BC) by default if no position is given
+fig.colorbar(cmap="roma", frame=["+Lvelocity", "y+lm/s"])
```

Comment:
Why are you using `+Lvelocity`? `x+lvelocity` make more sense to me.
```suggestion
fig.colorbar(cmap="roma", frame=["x+lvelocity", "y+lm/s"])
```

Reply:
Hmm you're right, must have got it from some old GMT example before.